docs/app/Examples/collections/Form/FieldVariations/FormFieldVariationsExamples.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+import ExampleSection from 'docs/app/Components/ComponentDoc/ExampleSection';
+import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample';
+
+export default class FormFieldVariationsExamples extends Component {
+  render() {
+    return (
+      <ExampleSection title='Field Variations'>
+        <ComponentExample
+          title='Inline'
+          description='A field can have its label next to instead of above it.'
+          examplePath='collections/Form/FieldVariations/FormFieldInlineExample'
+        />
+      </ExampleSection>
```

Comment:
Forms can be large or small!? wat?!

Oh here's the docs for it, http://semantic-ui.com/collections/form.html#size